help/develop/admin/quality-assurance.md

```diff
@@ -2,134 +2,172 @@
+Additionally, we use manual tests, automatic tests, monitoring and logging
+systems, and our [community](https://community.datagrok.ai/) to gather feedback
+and identify issues.
+
+You can [report an issue](../how-to/report-tickets.md) directly in one of our issue
+tracking systems. We use [JIRA](https://reddata.atlassian.net/) for internal
+issues, integrated with BitBucket, and [GitHub
+Tracker](https://github.com/datagrok-ai/public/issues) for all externally
+reported issues. We recommend reporting issues in JIRA if you have access to it,
+but if you don't, use GitHub Tracker.
+
+Tracking the issues, we stay on
+top of the platform's current status, including critical issues, promised
+features, and upcoming changes. You can also track our latest updates in [Release notes](../admin/releases/release-history.md).
+
+## QA tools
+
+### GrokTester
+
+Is an integrated monitoring system that runs automated package tests every hour. It's also possible to run [package testing locally](../how-to/test-packages#local-testing) before publishing.
 
 ### Test tracking system
 
-In order to keep track of test execution results, we have built a unique application on top of the Datagrok platform. It
-lets us not only visualize the current state of the application but also allows the QA engineer to efficiently work with
-stories by having the ability to quickly read them, execute, and report the completion status. Moreover, it lets us
-quickly navigate to the corresponding JIRA issues (integration with JIRA was done using Datagrok's OpenAPI capabilities)
-.
+To monitor the results of test execution, we have developed a unique
+application on top of the Datagrok platform. This application visualizes the application's current state and enables QA engineers
+promptly access, execute, and report the completion status of stories.
+Additionally, it lets us quickly navigate to the corresponding JIRA issues as we
+have integrated JIRA with Datagrok's OpenAPI capabilities.
 
 ![Test Tracking System](test-tracking-system.png)
 
-### Performance benchmarks
-
-We consider performance a critical feature of Datagrok, so we keep track of it. We run performance benchmarks as part of
-the build, and keep track of the results to make sure the platform only gets better with time.
-
-### Stress testing
-
-To make sure the platform is stable under heavy load (either many users working simultaneously or executing
-CPU-intensive computations), we perform automated
-[stress testing](https://en.wikipedia.org/wiki/Stress_testing)
-regularly. The real workflow is emulated by spawning many virtual machines that have Selenium installed on them and
-executing selected [UI tests](#ui-tests) on them. There are two different passes.
+### Test manager
 
-In the first pass, an ordinary yet intensive user activity is emulated by random-ranging tests.
+Is a tool within the Datagrok platform that provides a convenient interface for
```

Comment:
We shouldn't start paragraphs with "Is a tool" or "It is the". I think it should be "Test Manager is a tool ..."